app/routes/home.tsx

```
@@ -1,43 +1,101 @@
 import ResumeCard from "~/components/ResumeCard";
-import { resumes } from "../../constants";
-import { useNavigate } from "react-router";
-import { usePuterStore } from "~/lib/puter";
-import { useEffect } from "react";
+import {usePuterStore} from "~/lib/puter";
+import {Link} from "react-router";
+import {useEffect, useState} from "react";
+import { resumes as defaultResumes } from "../../constants";
+import { fadeIn } from "~/lib/utils";
 
 export function meta({}: Route.MetaArgs) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)